### DISCUSSION TITLE
Rows should only include panels explicitly added

### DISCUSSION THREAD

**Discussion by nabelekt** [grafana/grafana#64471] [Dashboards, Panels & Visualizations]
I want to reopen the issue that was closed at https://github.com/grafana/grafana/issues/33505 because "this has always been how row works." I am struggling to see how just about anyone who uses rows, or would like to use rows, wouldn't find the current implementation frustrating.

The frustration is that when you collapse a row, it garbles up all panels below it. This is exacerbated by the fact that when you add a new row, it is added to the top of the dashboard (https://github.com/grafana/grafana/discussions/38218), so anytime you add a row, it suddenly includes all panels that are on the dashboard, down to any pre-existing rows.

Rows should only include panels that have been explicitly dragged into it.

**Comment by Ray-Barker** [comment]
I found this frustrating from the get-go when I created a row in Grafana for the first time, my immediate thought was "what the heck is this behavior". And it was with Grafana v5.x. Here we are some years later with v10 and it's still here?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82587. Feel free to continue the discussion around this there. Thank you!
